run/temperature_monitor

```diff
@@ -73,16 +73,18 @@ do
     fi
   fi
 
-  # Log right after Archive_Clips
-  if [[ ( "${TEMPERATURE_POSTARCHIVE:-false}" = "true" ) && ( -e /tmp/flag_post_archive ) ]]
+  # Clear max temp and log right after Archive_Clips
+  if [[ -e /tmp/flag_post_archive ]]
   then
-    rm /tmp/flag_post_archive # Clear flag
-    message="Temperature Monitor: $(readable "$raw_temp") (max=$(readable "$peak_temp") $peak_time)"
-    /root/bin/send-push-message "$NOTIFICATION_TITLE:" "$message" || log "Temperature Monitor: Failed to send push message."
-
-    # Reset peak for new period after archiving
     peak_temp=$raw_temp
     peak_time=$timestamp
+
+    if [[ "${TEMPERATURE_POSTARCHIVE:-false}" = "true" ]]
+    then
+      rm /tmp/flag_post_archive # Clear flag
+      message="Temperature Monitor: $(readable "$raw_temp") (max=$(readable "$peak_temp") $peak_time)"
```

Comment:
This will now always show the same temperature for peak and raw temps. Did you mean to do that, and if so why?